config.fullstack.test.yaml

```diff
@@ -197,6 +197,9 @@ spec_root: &spec_root
   title: "The RESTBase root"
   # Some more general RESTBase info
   x-request-filters:
+    - path: lib/redirect_filter.js
+      options:
+        ar.wikipedia.beta.wmflabs.org: en.wikipedia.beta.wmflabs.org
```

Comment:
Would probably be nice to wrap this into a 'domain_redirects' property - just in case we ever want to add some other type of redirects.